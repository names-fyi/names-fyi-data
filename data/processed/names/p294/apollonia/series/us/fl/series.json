[{"y":1984,"t":6,"m":0,"f":6,"mr":null,"fr":1006,"mp":0,"fp":0.00009729677136880341},{"y":1985,"t":6,"m":0,"f":6,"mr":null,"fr":1068,"mp":0,"fp":0.00009193993257738278}]