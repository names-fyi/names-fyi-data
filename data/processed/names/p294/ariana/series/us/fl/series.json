[{"y":1978,"t":6,"m":0,"f":6,"mr":null,"fr":849,"mp":0,"fp":0.00013627381952803833},{"y":1980,"t":10,"m":0,"f":10,"mr":null,"fr":651,"mp":0,"fp":0.00019370835270416862},{"y":1981,"t":16,"m":0,"f":16,"mr":null,"fr":477,"mp":0,"fp":0.0002957704821058858},{"y":1982,"t":10,"m":0,"f":10,"mr":null,"fr":664,"mp":0,"fp":0.00017635751194822142},{"y":1983,"t":8,"m":0,"f":8,"mr":null,"fr":784,"mp":0,"fp":0.0001359342078434038},{"y":1984,"t":13,"m":0,"f":13,"mr":null,"fr":576,"mp":0,"fp":0.00021080967129907406},{"y":1985,"t":13,"m":0,"f":13,"mr":null,"fr":613,"mp":0,"fp":0.00019920318725099602},{"y":1986,"t":16,"m":0,"f":16,"mr":null,"fr":534,"mp":0,"fp":0.00024080428631629642},{"y":1987,"t":16,"m":0,"f":16,"mr":null,"fr":541,"mp":0,"fp":0.00023051433511021467},{"y":1988,"t":29,"m":0,"f":29,"mr":null,"fr":379,"mp":0,"fp":0.00039779429919618116},{"y":1989,"t":30,"m":0,"f":30,"mr":null,"fr":376,"mp":0,"fp":0.00038952944842630103},{"y":1990,"t":47,"m":0,"f":47,"mr":null,"fr":265,"mp":0,"fp":0.0005940869389354467},{"y":1991,"t":55,"m":0,"f":55,"mr":null,"fr":228,"mp":0,"fp":0.0007204233469558839},{"y":1992,"t":60,"m":0,"f":60,"mr":null,"fr":209,"mp":0,"fp":0.0007993711613530689},{"y":1993,"t":58,"m":0,"f":58,"mr":null,"fr":217,"mp":0,"fp":0.0007789835607607177},{"y":1994,"t":92,"m":0,"f":92,"mr":null,"fr":152,"mp":0,"fp":0.0012346341725267057},{"y":1995,"t":97,"m":0,"f":97,"mr":null,"fr":148,"mp":0,"fp":0.0013313386129373173},{"y":1996,"t":96,"m":0,"f":96,"mr":null,"fr":150,"mp":0,"fp":0.0013201138598204095},{"y":1997,"t":129,"m":0,"f":129,"mr":null,"fr":125,"mp":0,"fp":0.0017587631395967116},{"y":1998,"t":136,"m":0,"f":136,"mr":null,"fr":122,"mp":0,"fp":0.0018311565908172883},{"y":1999,"t":129,"m":0,"f":129,"mr":null,"fr":127,"mp":0,"fp":0.001726792048724985},{"y":2000,"t":184,"m":0,"f":184,"mr":null,"fr":90,"mp":0,"fp":0.0023804595321879527},{"y":2001,"t":210,"m":0,"f":210,"mr":null,"fr":78,"mp":0,"fp":0.0027157045313469896},{"y":2002,"t":174,"m":0,"f":174,"mr":null,"fr":89,"mp":0,"fp":0.002259388147301719},{"y":2003,"t":258,"m":0,"f":258,"mr":null,"fr":59,"mp":0,"fp":0.003232353604450124},{"y":2004,"t":236,"m":0,"f":236,"mr":null,"fr":70,"mp":0,"fp":0.002878189179959998},{"y":2005,"t":279,"m":0,"f":279,"mr":null,"fr":57,"mp":0,"fp":0.0032556945481702765},{"y":2006,"t":290,"m":0,"f":290,"mr":null,"fr":57,"mp":0,"fp":0.0032546238103788832},{"y":2007,"t":314,"m":0,"f":314,"mr":null,"fr":43,"mp":0,"fp":0.003517852541480411},{"y":2008,"t":250,"m":0,"f":250,"mr":null,"fr":62,"mp":0,"fp":0.002905557750865856},{"y":2009,"t":215,"m":0,"f":215,"mr":null,"fr":71,"mp":0,"fp":0.002612742893946943},{"y":2010,"t":242,"m":0,"f":242,"mr":null,"fr":57,"mp":0,"fp":0.003042570846639342},{"y":2011,"t":228,"m":0,"f":228,"mr":null,"fr":60,"mp":0,"fp":0.002868285318908039},{"y":2012,"t":232,"m":0,"f":232,"mr":null,"fr":58,"mp":0,"fp":0.0029159282581099253},{"y":2013,"t":295,"m":0,"f":295,"mr":null,"fr":38,"mp":0,"fp":0.0036382472281489336},{"y":2014,"t":365,"m":0,"f":365,"mr":null,"fr":27,"mp":0,"fp":0.004362845292310635},{"y":2015,"t":329,"m":0,"f":329,"mr":null,"fr":38,"mp":0,"fp":0.0038311945408389036},{"y":2016,"t":313,"m":0,"f":313,"mr":null,"fr":39,"mp":0,"fp":0.0036137345001962732},{"y":2017,"t":265,"m":0,"f":265,"mr":null,"fr":50,"mp":0,"fp":0.0030871028995468367},{"y":2018,"t":267,"m":0,"f":267,"mr":null,"fr":46,"mp":0,"fp":0.0031435433738344165},{"y":2019,"t":268,"m":0,"f":268,"mr":null,"fr":45,"mp":0,"fp":0.003173174832461105},{"y":2020,"t":244,"m":0,"f":244,"mr":null,"fr":50,"mp":0,"fp":0.00304046055500866},{"y":2021,"t":225,"m":0,"f":225,"mr":null,"fr":58,"mp":0,"fp":0.0026988125224901042},{"y":2022,"t":198,"m":0,"f":198,"mr":null,"fr":74,"mp":0,"fp":0.002312057731380929}]