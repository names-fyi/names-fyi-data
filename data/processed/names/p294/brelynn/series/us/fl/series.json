[{"y":2015,"t":6,"m":0,"f":6,"mr":null,"fr":1880,"mp":0,"fp":0.00006986980925542074},{"y":2016,"t":6,"m":0,"f":6,"mr":null,"fr":1850,"mp":0,"fp":0.0000692728658184589},{"y":2020,"t":5,"m":0,"f":5,"mr":null,"fr":2059,"mp":0,"fp":0.00006230451956984959}]