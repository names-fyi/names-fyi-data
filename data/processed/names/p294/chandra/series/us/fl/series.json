[{"y":1956,"t":11,"m":0,"f":11,"mr":null,"fr":415,"mp":0,"fp":0.00025581990278843696},{"y":1957,"t":7,"m":0,"f":7,"mr":null,"fr":582,"mp":0,"fp":0.00015244898403641354},{"y":1958,"t":13,"m":0,"f":13,"mr":null,"fr":425,"mp":0,"fp":0.00027401880190548457},{"y":1959,"t":5,"m":0,"f":5,"mr":null,"fr":744,"mp":0,"fp":0.00010243382774727526},{"y":1960,"t":8,"m":0,"f":8,"mr":null,"fr":587,"mp":0,"fp":0.00015917863822674996},{"y":1961,"t":14,"m":0,"f":14,"mr":null,"fr":434,"mp":0,"fp":0.00027970351427486864},{"y":1962,"t":8,"m":0,"f":8,"mr":null,"fr":614,"mp":0,"fp":0.00016259171188748653},{"y":1963,"t":6,"m":0,"f":6,"mr":null,"fr":719,"mp":0,"fp":0.00012367564002143712},{"y":1964,"t":13,"m":0,"f":13,"mr":null,"fr":469,"mp":0,"fp":0.000270714895566523},{"y":1965,"t":16,"m":0,"f":16,"mr":null,"fr":390,"mp":0,"fp":0.00036032789838753263},{"y":1966,"t":11,"m":0,"f":11,"mr":null,"fr":481,"mp":0,"fp":0.0002565119044842945},{"y":1967,"t":9,"m":0,"f":9,"mr":null,"fr":530,"mp":0,"fp":0.00021734405564007825},{"y":1968,"t":28,"m":0,"f":28,"mr":null,"fr":270,"mp":0,"fp":0.0006627846423330019},{"y":1969,"t":31,"m":0,"f":31,"mr":null,"fr":277,"mp":0,"fp":0.0006985128436232537},{"y":1970,"t":28,"m":0,"f":28,"mr":null,"fr":294,"mp":0,"fp":0.0005880499842486611},{"y":1971,"t":31,"m":0,"f":31,"mr":null,"fr":282,"mp":0,"fp":0.0006491602797671399},{"y":1972,"t":28,"m":0,"f":28,"mr":null,"fr":287,"mp":0,"fp":0.0006293549112160036},{"y":1973,"t":40,"m":0,"f":40,"mr":null,"fr":220,"mp":0,"fp":0.0009312937999115271},{"y":1974,"t":34,"m":0,"f":34,"mr":null,"fr":254,"mp":0,"fp":0.0007779786284694414},{"y":1975,"t":31,"m":0,"f":31,"mr":null,"fr":255,"mp":0,"fp":0.0007496977025392987},{"y":1976,"t":30,"m":0,"f":30,"mr":null,"fr":256,"mp":0,"fp":0.000732171621027969},{"y":1977,"t":27,"m":0,"f":27,"mr":null,"fr":285,"mp":0,"fp":0.0006249132064990974},{"y":1978,"t":27,"m":0,"f":27,"mr":null,"fr":281,"mp":0,"fp":0.0006132321878761725},{"y":1979,"t":25,"m":0,"f":25,"mr":null,"fr":316,"mp":0,"fp":0.0005333333333333334},{"y":1980,"t":26,"m":0,"f":26,"mr":null,"fr":321,"mp":0,"fp":0.0005036417170308383},{"y":1981,"t":24,"m":0,"f":24,"mr":null,"fr":350,"mp":0,"fp":0.00044365572315882877},{"y":1982,"t":23,"m":0,"f":23,"mr":null,"fr":380,"mp":0,"fp":0.0004056222774809093},{"y":1983,"t":13,"m":0,"f":13,"mr":null,"fr":575,"mp":0,"fp":0.00022089308774553117},{"y":1984,"t":20,"m":0,"f":20,"mr":null,"fr":441,"mp":0,"fp":0.0003243225712293447},{"y":1985,"t":15,"m":0,"f":15,"mr":null,"fr":562,"mp":0,"fp":0.00022984983144345693},{"y":1986,"t":13,"m":0,"f":13,"mr":null,"fr":634,"mp":0,"fp":0.00019565348263199086},{"y":1987,"t":20,"m":0,"f":20,"mr":null,"fr":463,"mp":0,"fp":0.00028814291888776836},{"y":1988,"t":13,"m":0,"f":13,"mr":null,"fr":683,"mp":0,"fp":0.0001783215823982881},{"y":1989,"t":21,"m":0,"f":21,"mr":null,"fr":505,"mp":0,"fp":0.0002726706138984107},{"y":1990,"t":13,"m":0,"f":13,"mr":null,"fr":730,"mp":0,"fp":0.0001643219192800172},{"y":1991,"t":10,"m":0,"f":10,"mr":null,"fr":910,"mp":0,"fp":0.000130986063082888},{"y":1992,"t":7,"m":0,"f":7,"mr":null,"fr":1150,"mp":0,"fp":0.00009325996882452471},{"y":1993,"t":8,"m":0,"f":8,"mr":null,"fr":1033,"mp":0,"fp":0.00010744600838078866}]