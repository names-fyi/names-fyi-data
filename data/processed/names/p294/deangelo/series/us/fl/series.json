[{"y":1973,"t":6,"m":6,"f":0,"mr":626,"fr":null,"mp":0.00012116071969467499,"fp":0},{"y":1974,"t":5,"m":5,"f":0,"mr":706,"fr":null,"mp":0.00009867966606801003,"fp":0},{"y":1975,"t":5,"m":5,"f":0,"mr":686,"fr":null,"mp":0.00010417534794566213,"fp":0},{"y":1976,"t":5,"m":5,"f":0,"mr":699,"fr":null,"mp":0.00010593669223272173,"fp":0},{"y":1977,"t":11,"m":11,"f":0,"mr":448,"fr":null,"mp":0.00021984171396594452,"fp":0},{"y":1978,"t":6,"m":6,"f":0,"mr":662,"fr":null,"mp":0.00011755485893416928,"fp":0},{"y":1979,"t":5,"m":5,"f":0,"mr":761,"fr":null,"mp":0.00009131752931292691,"fp":0},{"y":1980,"t":11,"m":11,"f":0,"mr":501,"fr":null,"mp":0.0001840111075795848,"fp":0},{"y":1982,"t":6,"m":6,"f":0,"mr":736,"fr":null,"mp":0.00009079765742043855,"fp":0},{"y":1984,"t":14,"m":14,"f":0,"mr":469,"fr":null,"mp":0.00019656295630686286,"fp":0},{"y":1985,"t":11,"m":11,"f":0,"mr":559,"fr":null,"mp":0.00014740170985983437,"fp":0},{"y":1986,"t":14,"m":14,"f":0,"mr":489,"fr":null,"mp":0.0001832053077194865,"fp":0},{"y":1987,"t":14,"m":14,"f":0,"mr":513,"fr":null,"mp":0.00017607405171546434,"fp":0},{"y":1988,"t":15,"m":15,"f":0,"mr":519,"fr":null,"mp":0.0001776598642678637,"fp":0},{"y":1989,"t":22,"m":22,"f":0,"mr":438,"fr":null,"mp":0.0002463247231646009,"fp":0},{"y":1990,"t":27,"m":27,"f":0,"mr":402,"fr":null,"mp":0.00029450583012467416,"fp":0},{"y":1991,"t":10,"m":10,"f":0,"mr":716,"fr":null,"mp":0.00011235702568481607,"fp":0},{"y":1992,"t":16,"m":16,"f":0,"mr":560,"fr":null,"mp":0.0001828759529551611,"fp":0},{"y":1993,"t":10,"m":10,"f":0,"mr":740,"fr":null,"mp":0.00011391597557641484,"fp":0},{"y":1994,"t":12,"m":12,"f":0,"mr":668,"fr":null,"mp":0.00014046916701783958,"fp":0},{"y":1995,"t":12,"m":12,"f":0,"mr":675,"fr":null,"mp":0.00014137772593927827,"fp":0},{"y":1996,"t":27,"m":27,"f":0,"mr":421,"fr":null,"mp":0.0003181373646443342,"fp":0},{"y":1997,"t":17,"m":17,"f":0,"mr":565,"fr":null,"mp":0.00019798751514022174,"fp":0},{"y":1998,"t":17,"m":17,"f":0,"mr":590,"fr":null,"mp":0.0001961417758907144,"fp":0},{"y":1999,"t":19,"m":19,"f":0,"mr":546,"fr":null,"mp":0.00021883098186006335,"fp":0},{"y":2000,"t":14,"m":14,"f":0,"mr":682,"fr":null,"mp":0.0001568065231513631,"fp":0},{"y":2001,"t":11,"m":11,"f":0,"mr":810,"fr":null,"mp":0.00012248215657673507,"fp":0},{"y":2002,"t":17,"m":17,"f":0,"mr":615,"fr":null,"mp":0.00018895607327049617,"fp":0},{"y":2003,"t":10,"m":10,"f":0,"mr":901,"fr":null,"mp":0.0001070056605994457,"fp":0},{"y":2004,"t":13,"m":13,"f":0,"mr":783,"fr":null,"mp":0.00013598184119412976,"fp":0},{"y":2005,"t":22,"m":22,"f":0,"mr":590,"fr":null,"mp":0.0002222738616042111,"fp":0},{"y":2006,"t":20,"m":20,"f":0,"mr":664,"fr":null,"mp":0.00019353215536761434,"fp":0},{"y":2007,"t":15,"m":15,"f":0,"mr":798,"fr":null,"mp":0.00014403134121984944,"fp":0},{"y":2008,"t":25,"m":25,"f":0,"mr":582,"fr":null,"mp":0.0002488156375651897,"fp":0},{"y":2009,"t":18,"m":18,"f":0,"mr":706,"fr":null,"mp":0.0001889684422701409,"fp":0},{"y":2010,"t":23,"m":23,"f":0,"mr":583,"fr":null,"mp":0.00024838012958963284,"fp":0},{"y":2011,"t":23,"m":23,"f":0,"mr":591,"fr":null,"mp":0.00025007611012047145,"fp":0},{"y":2012,"t":19,"m":19,"f":0,"mr":679,"fr":null,"mp":0.00020674871326129774,"fp":0},{"y":2013,"t":13,"m":13,"f":0,"mr":878,"fr":null,"mp":0.0001393086006986862,"fp":0},{"y":2014,"t":17,"m":17,"f":0,"mr":764,"fr":null,"mp":0.00017723287357040835,"fp":0},{"y":2015,"t":15,"m":15,"f":0,"mr":847,"fr":null,"mp":0.0001526577718071627,"fp":0},{"y":2016,"t":8,"m":8,"f":0,"mr":1257,"fr":null,"mp":0.00008177617859917406,"fp":0},{"y":2017,"t":18,"m":18,"f":0,"mr":740,"fr":null,"mp":0.00018645121193287757,"fp":0},{"y":2018,"t":13,"m":13,"f":0,"mr":946,"fr":null,"mp":0.00013584542880132084,"fp":0},{"y":2019,"t":15,"m":15,"f":0,"mr":870,"fr":null,"mp":0.00015859086728058953,"fp":0},{"y":2020,"t":12,"m":12,"f":0,"mr":1001,"fr":null,"mp":0.00013333925952264546,"fp":0},{"y":2021,"t":10,"m":10,"f":0,"mr":1155,"fr":null,"mp":0.00010812096573646596,"fp":0},{"y":2022,"t":8,"m":8,"f":0,"mr":1369,"fr":null,"mp":0.00008443004443131089,"fp":0}]