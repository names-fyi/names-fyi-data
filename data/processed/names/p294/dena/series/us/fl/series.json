[{"y":1951,"t":5,"m":0,"f":5,"mr":null,"fr":533,"mp":0,"fp":0.0001621060822202049},{"y":1954,"t":9,"m":0,"f":9,"mr":null,"fr":431,"mp":0,"fp":0.00023901843097678865},{"y":1955,"t":11,"m":0,"f":11,"mr":null,"fr":400,"mp":0,"fp":0.0002776866180294348},{"y":1956,"t":9,"m":0,"f":9,"mr":null,"fr":474,"mp":0,"fp":0.00020930719319053932},{"y":1957,"t":9,"m":0,"f":9,"mr":null,"fr":507,"mp":0,"fp":0.00019600583661824597},{"y":1958,"t":17,"m":0,"f":17,"mr":null,"fr":366,"mp":0,"fp":0.00035833227941486445},{"y":1959,"t":17,"m":0,"f":17,"mr":null,"fr":372,"mp":0,"fp":0.0003482750143407359},{"y":1960,"t":17,"m":0,"f":17,"mr":null,"fr":386,"mp":0,"fp":0.00033825460623184366},{"y":1961,"t":30,"m":0,"f":30,"mr":null,"fr":276,"mp":0,"fp":0.0005993646734461471},{"y":1962,"t":25,"m":0,"f":25,"mr":null,"fr":304,"mp":0,"fp":0.0005080990996483955},{"y":1963,"t":30,"m":0,"f":30,"mr":null,"fr":281,"mp":0,"fp":0.0006183782001071855},{"y":1964,"t":22,"m":0,"f":22,"mr":null,"fr":347,"mp":0,"fp":0.00045813290018950045},{"y":1965,"t":28,"m":0,"f":28,"mr":null,"fr":280,"mp":0,"fp":0.0006305738221781821},{"y":1966,"t":24,"m":0,"f":24,"mr":null,"fr":297,"mp":0,"fp":0.0005596623370566425},{"y":1967,"t":31,"m":0,"f":31,"mr":null,"fr":250,"mp":0,"fp":0.0007486295249824917},{"y":1968,"t":29,"m":0,"f":29,"mr":null,"fr":267,"mp":0,"fp":0.0006864555224163234},{"y":1969,"t":36,"m":0,"f":36,"mr":null,"fr":238,"mp":0,"fp":0.0008111762054979721},{"y":1970,"t":53,"m":0,"f":53,"mr":null,"fr":183,"mp":0,"fp":0.0011130946130421086},{"y":1971,"t":48,"m":0,"f":48,"mr":null,"fr":202,"mp":0,"fp":0.001005151400929765},{"y":1972,"t":30,"m":0,"f":30,"mr":null,"fr":277,"mp":0,"fp":0.0006743088334457181},{"y":1973,"t":25,"m":0,"f":25,"mr":null,"fr":305,"mp":0,"fp":0.0005820586249447045},{"y":1974,"t":35,"m":0,"f":35,"mr":null,"fr":244,"mp":0,"fp":0.0008008603528361898},{"y":1975,"t":20,"m":0,"f":20,"mr":null,"fr":355,"mp":0,"fp":0.0004836759371221282},{"y":1976,"t":18,"m":0,"f":18,"mr":null,"fr":366,"mp":0,"fp":0.00043930297261678136},{"y":1977,"t":16,"m":0,"f":16,"mr":null,"fr":432,"mp":0,"fp":0.00037031893718465027},{"y":1978,"t":20,"m":0,"f":20,"mr":null,"fr":356,"mp":0,"fp":0.0004542460650934611},{"y":1979,"t":22,"m":0,"f":22,"mr":null,"fr":347,"mp":0,"fp":0.0004693333333333333},{"y":1980,"t":21,"m":0,"f":21,"mr":null,"fr":385,"mp":0,"fp":0.0004067875406787541},{"y":1981,"t":17,"m":0,"f":17,"mr":null,"fr":463,"mp":0,"fp":0.0003142561372375037},{"y":1982,"t":12,"m":0,"f":12,"mr":null,"fr":598,"mp":0,"fp":0.0002116290143378657},{"y":1983,"t":19,"m":0,"f":19,"mr":null,"fr":446,"mp":0,"fp":0.000322843743628084},{"y":1984,"t":24,"m":0,"f":24,"mr":null,"fr":376,"mp":0,"fp":0.00038918708547521364},{"y":1985,"t":13,"m":0,"f":13,"mr":null,"fr":622,"mp":0,"fp":0.00019920318725099602},{"y":1986,"t":12,"m":0,"f":12,"mr":null,"fr":675,"mp":0,"fp":0.00018060321473722233},{"y":1987,"t":8,"m":0,"f":8,"mr":null,"fr":909,"mp":0,"fp":0.00011525716755510733},{"y":1988,"t":11,"m":0,"f":11,"mr":null,"fr":766,"mp":0,"fp":0.00015088749279855147},{"y":1989,"t":11,"m":0,"f":11,"mr":null,"fr":826,"mp":0,"fp":0.00014282746442297704},{"y":1990,"t":6,"m":0,"f":6,"mr":null,"fr":1285,"mp":0,"fp":0.0000758408858215464},{"y":1991,"t":6,"m":0,"f":6,"mr":null,"fr":1304,"mp":0,"fp":0.00007859163784973278},{"y":1992,"t":5,"m":0,"f":5,"mr":null,"fr":1495,"mp":0,"fp":0.00006661426344608908},{"y":1995,"t":7,"m":0,"f":7,"mr":null,"fr":1156,"mp":0,"fp":0.0000960759823769198},{"y":1997,"t":6,"m":0,"f":6,"mr":null,"fr":1367,"mp":0,"fp":0.00008180293672542844},{"y":2000,"t":5,"m":0,"f":5,"mr":null,"fr":1760,"mp":0,"fp":0.00006468640033119437},{"y":2002,"t":7,"m":0,"f":7,"mr":null,"fr":1377,"mp":0,"fp":0.00009089492546616112},{"y":2003,"t":6,"m":0,"f":6,"mr":null,"fr":1644,"mp":0,"fp":0.00007517101405697962}]