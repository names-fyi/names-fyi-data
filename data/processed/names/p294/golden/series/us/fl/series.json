[{"y":2018,"t":5,"m":0,"f":5,"mr":null,"fr":2149,"mp":0,"fp":0.000058867853442592066},{"y":2022,"t":6,"m":6,"f":0,"mr":1669,"fr":null,"mp":0.00006332253332348316,"fp":0}]